digdag-standards/src/main/java/io/digdag/standards/command/ecs/DefaultEcsClient.java

```diff
@@ -270,6 +270,8 @@ public GetLogEventsResult getLog(
             final Optional<String> nextToken)
     {
         final GetLogEventsRequest request = new GetLogEventsRequest()
+                // This should be true when using `nextToken`. See the doc for details.
+                .withStartFromHead(true)
```

Comment:
`If StartFromHead is true, the earliest log events are returned first.` and StartFromHead = false by the default.

Is this means console output will become reverse order?

Reply:
Thanks for the testing. According to the test result you shared, `startFromHead: true` may affect only if we would specify `limit`. 

You mean we'd better sort `events` list just in case?